DuckDuckGo/OnboardingExperiment/DaxDialogs/DaxDialogView.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  DaxDialogView.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+import DesignResourcesKit
+
+// MARK: - Metrics
+
+private enum Metrics {
+    static let darkBackground = Color(0x2F2F2F, alpha: 1)
+    static let whiteBackground = Color.white
+    static let contentPadding: CGFloat = 24.0
+    static let shadowRadius: CGFloat = 5.0
+    static let stackSpacing: CGFloat = 10.0
+
+    enum DaxLogo {
+        static let size: CGFloat = 54.0
+        static let horizontalPadding: CGFloat = 10
+    }
+}
+
+// MARK: - DaxDialog
+
+struct DaxDialogView<Content: View>: View {
+
+    enum LogoPosition {
+        case top
+        case left
+    }
+
+    @Environment(\.colorScheme) var colorScheme
+
+    @State private var logoPosition: LogoPosition
+    private let cornerRadius: CGFloat
+    private let arrowSize: CGSize
+    private let content: Content
+
+    init(
+        logoPosition: LogoPosition,
+        cornerRadius: CGFloat = 16.0,
+        arrowSize: CGSize = .init(width: 16.0, height: 8.0),
+        @ViewBuilder content: () -> Content
+    ) {
+        _logoPosition = State(initialValue: logoPosition)
+        self.cornerRadius = cornerRadius
+        self.arrowSize = arrowSize
+        self.content = content()
+    }
+
+    var body: some View {
+        switch logoPosition {
+        case .top:
+            topLogoViewContentView
+        case .left:
+            leftLogoContentView
+        }
+    }
+
+    private var topLogoViewContentView: some View {
+        VStack(alignment: .leading, spacing: Metrics.stackSpacing) {
+            daxLogo
+                .padding(.leading, Metrics.DaxLogo.horizontalPadding)
+
+            wrappedContent
+        }
+    }
+
+    private var leftLogoContentView: some View {
+        HStack(alignment: .top, spacing: Metrics.stackSpacing) {
+            daxLogo
+
+            wrappedContent
+        }
+
+    }
+
+    private var daxLogo: some View {
+        Image(.daxIcon)
+            .resizable()
+            .aspectRatio(contentMode: .fill)
+            .frame(width: Metrics.DaxLogo.size, height: Metrics.DaxLogo.size)
+    }
+
+    private var wrappedContent: some View {
+        let backgroundColor = colorScheme == .light ? Metrics.whiteBackground : Metrics.darkBackground
```

Comment:
See note above about using asset catalog or design system colour directly.

Reply:
found the color thanks!